src/raw.rs

```diff
@@ -29,6 +29,17 @@ bitflags! {
     pub struct KeyMode: c_int {
         const READ = REDISMODULE_READ as c_int;
         const WRITE = REDISMODULE_WRITE as c_int;
+
+        /// Avoid touching the LRU/LFU of the key when opened.
```

Comment:
We said we will split it to a separate struct no?

Reply:
Any Ideas how to `|` between other types? Let's slack this one?